### ISSUE TITLE
Docker container

### ISSUE THREAD

**Issue by vetash** [EXG1O/Constructor-Telegram-Bots#109]
Hello!  Awesome project!

Can you provide instructions to create docker container? Or mb pub one on docker hub?

Offtop:
На каком языке лучше создавать issue?)

**Comment by EXG1O** [comment]
Привет, issue лучше всего создавать на русском языке.
По поводу Docker'а не понял в чём у тебя проблема.

**Comment by baha-code** [comment]
Здравствуйте! когда будет проект в Docker?

**Comment by EXG1O** [comment]
Здравствуйте, точно не могу сказать, потому-что сейчас я занимаюсь реконструкции проекта + переход с JS на TS + внедрение разных технологий, который помогут проекту в будущем или уже + добавления нового функционала и улучшение существующего. Так что ожидайте в скором времени крупного обновления, так как я уже работаю над ним больше месяца.

**Comment by Impramine** [comment]
Здравствуйте, не готов ли еще docker контейнер?

**Comment by EXG1O** [comment]
> Здравствуйте, не готов ли еще docker контейнер?

Здравствуйте, к сожалению нет. Сейчас я работаю уже 8ый месяц над очень крупным обновлениям, которое выйдет в ближайших числах, и после его выхода я займусь Docker контейнером.

**Comment by KGBByeBye** [comment]
Очень ждём:)

**Comment by psaxys** [comment]
Хотелось бы обновление и контейнер Docker

**Comment by EXG1O** [comment]
> Хотелось бы обновление и контейнер Docker

Скоро уже всё будет))

**Comment by EvgeniiItImperia** [comment]
Сервис потрясающий !!!!    
Из пожелания: очень не хватает функции копирования блоков  в конструкторе

**Comment by EXG1O** [comment]
> Сервис потрясающий !!!! Из пожелания: очень не хватает функции копирования блоков в конструкторе

Спасибо большое, мне очень приятно!
По поводу пожеланий, скоро выйдет очень крупное обновление))

**Comment by EvgeniiItImperia** [comment]
> > Сервис потрясающий !!!! Из пожелания: очень не хватает функции копирования блоков в конструкторе
> 
> Спасибо большое, мне очень приятно! По поводу пожеланий, скоро выйдет очень крупное обновление))

очень ждем !! ))  Только порадовался какой классный сервис, а оказывается если токен обновить в ТГ, бот удаляется с сервиса 😢  Печалька

**Comment by EXG1O** [comment]
> > > Сервис потрясающий !!!! Из пожелания: очень не хватает функции копирования блоков в конструкторе
> > 
> > 
> > Спасибо большое, мне очень приятно! По поводу пожеланий, скоро выйдет очень крупное обновление))
> 
> очень ждем !! )) Только порадовался какой классный сервис, а оказывается если токен обновить в ТГ, бот удаляется с сервиса 😢 Печалька

Да, в нынешней версии проекта есть такая проблема, но в следующих такого не будет, простите за потраченное время на добавление команд и т.п.
